### PR TITLE
Dynamic theming via CSS variables

### PR DESCRIPTION
**Pull request by dnlkoch**
## Description

This adds support for dynamic component theming via antd's [CSS variables](https://ant.design/docs/react/customize-theme-variable).

**Migration notes:**

- `react-geo` doesn't include the antd styles anymore and applications must include either `antd.variable.min.css` or `antd.variable.css`: 

```css
@import '~antd/dist/antd.variable.min.css';
```

Any other style imports (e.g. `antd.min.css`) should be removed.

- Existing overrides/adjustements to the less variables need to be transformed to the antd CSS variables, see [here](https://ant.design/docs/react/customize-theme-variable) for further details.

- Some former less variables of `react-geo` were removed and are available as CSS variables now:
  - `@circle-menu-background-color` --> `--circle-menu-background-color`
  - `@measure-static-tooltip-background-color` --> `--measure-static-tooltip-background-color`
  - `@measure-dynamic-tooltip-background-color` --> `--measure-dynamic-tooltip-background-color`

Please review @terrestris/devs.

## Related issues or pull requests

--

## Pull request type

- [ ] Bugfix
- [x] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Do you introduce a breaking change?

- [x] Yes
- [ ] No

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the
  [BSD-2-Clause](https://github.com/terrestris/react-geo/blob/main/LICENSE).
- [x] I have followed the [guidelines for contributing](https://github.com/terrestris/react-geo/blob/main/CONTRIBUTING.md).
- [x] The proposed change fits to the content of the [Code of Conduct](https://github.com/terrestris/react-geo/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally).
- [ ] I have added a screenshot/screencast to illustrate the visual output of my update.

### CODE REVIEW

File: src/Panel/Panel/Panel.less [hunk 1]
```diff
@@ -1,12 +1,10 @@
-@import '../../style/variables.less';
-
-@handlestyle: 5px solid fade(@primary-color, 50%);
+@handlestyle: 5px solid var(--ant-primary-4);
```
Comment (KaiVolland) [inline src/Panel/Panel/Panel.less:1]: I'd suggest to transform this into a css custom property too.

File: src/Panel/Panel/Panel.less [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
 
 .react-geo-panel {
   z-index: 100;
 
   .body {
-    background-color: @component-background;
+    background-color: white;
```
Comment (KaiVolland) [inline src/Panel/Panel/Panel.less:7]: Is there no replacement for `@component-background`?

**Review by KaiVolland — APPROVED** [review]
LGTM. Just two notes.

File: src/Panel/Panel/Panel.less (hunk 2 above)
Comment (dnlkoch) [reply src/Panel/Panel/Panel.less:7]: No, only a small subset of the existing less variables is available yet.

File: src/Panel/Panel/Panel.less (hunk 1 above)
Comment (dnlkoch) [reply src/Panel/Panel/Panel.less:1]: Actually the value of `--ant-primary-4` is automatically generated based on the given primary color. Thus the value is more or less guaranteed to match the desired color scheme.

**Review by LukasLohoff — APPROVED** [review]

**Comment by dnlkoch** [comment]
Thanks for the reviews!

**Pull request merged**